modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletGetPropsTest.java

```diff
@@ -0,0 +1,1138 @@
+
+			JSONArray vocabulariesJSONArray = (JSONArray)props.get(
+				"vocabularies");
+
+			Assert.assertEquals(
+				JSONUtil.putAll(
+					JSONUtil.put(
+						"categories",
+						JSONUtil.putAll(
+							JSONUtil.put(
+								"key",
+								String.valueOf(
+									childAssetCategory1.getCategoryId())
+							).put(
+								"name", "child-category-1"
+							).put(
+								"value", 1L
+							).put(
+								"vocabularyName", "child-vocabulary"
+							),
+							JSONUtil.put(
+								"key",
+								String.valueOf(
+									childAssetCategory2.getCategoryId())
+							).put(
+								"name", "child-category-2"
+							).put(
+								"value", 2L
+							).put(
+								"vocabularyName", "child-vocabulary"
+							))
+					).put(
+						"key", String.valueOf(assetCategory.getCategoryId())
+					).put(
+						"name", "category"
+					).put(
+						"value", 2L
+					).put(
+						"vocabularyName", "vocabulary"
+					)
+				).toString(),
+				vocabulariesJSONArray.toString());
+		}
+		finally {
+			_assetVocabularyLocalService.deleteVocabulary(assetVocabulary);
+			_assetVocabularyLocalService.deleteVocabulary(childAssetVocabulary);
+		}
+	}
+
+	@Test
+	public void testGetPropsWithChildNoneAssetCategory() throws Exception {
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext(
+				_company.getGroupId(), TestPropsValues.getUserId());
+
+		AssetVocabulary assetVocabulary =
+			_assetVocabularyLocalService.addVocabulary(
+				TestPropsValues.getUserId(), _company.getGroupId(),
+				"vocabulary", serviceContext);
+
+		AssetCategory assetCategory = _assetCategoryLocalService.addCategory(
+			TestPropsValues.getUserId(), _company.getGroupId(), "category",
+			assetVocabulary.getVocabularyId(), serviceContext);
+
+		AssetVocabulary childAssetVocabulary =
+			_assetVocabularyLocalService.addVocabulary(
+				TestPropsValues.getUserId(), _company.getGroupId(),
+				"child-vocabulary", serviceContext);
+
+		AssetCategory childAssetCategory =
+			_assetCategoryLocalService.addCategory(
+				TestPropsValues.getUserId(), _company.getGroupId(),
+				"child-category", childAssetVocabulary.getVocabularyId(),
+				serviceContext);
+
+		try {
+			JournalTestUtil.addArticle(
+				_group.getGroupId(), 0,
+				_getServiceContext(
+					TestPropsValues.getUserId(), _group.getGroupId(),
+					new long[] {assetCategory.getCategoryId()}));
+			JournalTestUtil.addArticle(
+				_group.getGroupId(), 0,
+				_getServiceContext(
+					TestPropsValues.getUserId(), _group.getGroupId(),
+					new long[] {
+						assetCategory.getCategoryId(),
+						childAssetCategory.getCategoryId()
+					}));
+
+			Map<String, Object> data = _getData(
+				_getMockLiferayPortletRenderRequest(
+					new String[] {
+						assetVocabulary.getName(),
+						childAssetVocabulary.getName()
+					}));
+
+			Map<String, Object> props = (Map<String, Object>)data.get("props");
+
+			Assert.assertNotNull(props);
```

Comment:
Not needed

Reply:
true! i forgot to do it, removing... Thanks!